src/Components/test/testassets/Ignitor/ElementNode.cs

```diff
@@ -63,14 +63,14 @@ public void SetEvent(string eventName, ElementEventDescriptor descriptor)
             _events[eventName] = descriptor;
         }
 
-        internal async Task SelectAsync(HubConnection connection, string value)
+        internal Task SelectAsync(HubConnection connection, string value)
         {
             if (!Events.TryGetValue("change", out var changeEventDescriptor))
             {
-                throw new InvalidOperationException("Element does not have a click event.");
+                throw new InvalidOperationException("Element does not have a change event.");
             }
 
-            var mouseEventArgs = new UIChangeEventArgs()
+            var sleectEventArgs = new UIChangeEventArgs()
```

Comment:
If we need further manual updates suggest fixing the typo in this name
```suggestion
            var selectEventArgs = new UIChangeEventArgs()
```